CMakeLists.txt

```diff
@@ -269,7 +269,7 @@ endif (UNIT_TEST)
 # Common include
 #
 include_directories("/usr/include")
-#include_directories("/usr/local/include/libbson-1.0")
+include_directories("/usr/local/include/libbson-1.0")
```

Comment:
Not sure what this change was made, as it should only be necessary for the monogc driver.
However, the compilation worked in Travis, so, let's keep it. It's a step in the right direction! :)

NTC